guides/micronaut-graphql-todo/java/src/main/java/example/micronaut/GraphQLFactory.java

```diff
@@ -0,0 +1,61 @@
+package example.micronaut;
+
+import graphql.GraphQL;
+import graphql.schema.GraphQLSchema;
+import graphql.schema.idl.RuntimeWiring;
+import graphql.schema.idl.SchemaGenerator;
+import graphql.schema.idl.SchemaParser;
+import graphql.schema.idl.TypeDefinitionRegistry;
+import graphql.schema.idl.errors.SchemaMissingError;
+import io.micronaut.context.annotation.Bean;
+import io.micronaut.context.annotation.Factory;
+import io.micronaut.core.io.ResourceResolver;
+import jakarta.inject.Singleton;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+@Factory
+public class GraphQLFactory {
+
+    @Bean
+    @Singleton
+    public GraphQL graphQL(ResourceResolver resourceResolver,
+                           ToDosDataFetcher toDosDataFetcher,
+                           CreateToDoDataFetcher createToDoDataFetcher,
+                           CompleteToDoDataFetcher completeToDoDataFetcher,
+                           AuthorDataFetcher authorDataFetcher) {
+        SchemaParser schemaParser = new SchemaParser();
+        SchemaGenerator schemaGenerator = new SchemaGenerator();
+
+        // Parse the schema.
+        InputStream schemaDefinition = resourceResolver
+                .getResourceAsStream("classpath:schema.graphqls")
+                .orElseThrow(SchemaMissingError::new);
+
+        // Merge it into a type registry
```

Comment:
"Merge it into a type registry" -> "Parse the schema and merge it into a type registry"